src/post/postActions.js

```diff
@@ -25,8 +26,11 @@ export const getContent = (
     payload: {
       promise: steemAPI
         .getContentAsync(postAuthor, postPermlink)
-        .then(postData => omit(postData, omitAttributes))
-    }
+        .then((postData) => {
```

Comment:
We probably should indent `then` to give visual clue that it belongs to `getContentAsync`